networkx/readwrite/gml.py

```diff
@@ -164,7 +164,20 @@ def read_gml(path, label="label", destringizer=None):
     >>> G = nx.path_graph(4)
     >>> nx.write_gml(G, "test.gml")
     >>> H = nx.read_gml("test.gml")
-    >>> I = nx.read_gml("test.gml", destringizer=int)
+
+    >>> I = nx.MultiGraph()
+    >>> I.add_edge(1, 3, data = {'s': 1})
+    0
+    >>> print(I.nodes)
+    [1, 3]
+    >>> nx.write_gml(I, "test.gml")
+    >>> J = nx.read_gml("test.gml")
+    >>> K = nx.read_gml("test.gml", destringizer=int)
+    >>> print(J.nodes)
+    ['1', '3']
+    >>> print(K.nodes)
+    [1, 3]
```

Comment:
Thanks @randallwvr90 , this is indeed what I had in mind.

I think a bit of explanation interspersed with the code will go a long way to making the destringizer example even more clear, e.g. something like:

```
    Examples                                                                    
    --------                                                                    
    >>> G = nx.path_graph(4)                                                    
    >>> G.nodes                                                                 
    NodeView((0, 1, 2, 3))                                                      
    >>> nx.write_gml(G, "test.gml")                                             
                                                                                
    GML values are interpreted as strings by default:                 
                                                                                
    >>> H = nx.read_gml("test.gml")                                             
    >>> H.nodes                                                                 
    NodeView(('0', '1', '2', '3'))                                              
                                                                                
    `destringizer` can be used to convert GML values to other Python objects.   
    For example, to recover integer nodes when reading:                         
                                                                                
    >>> J = nx.read_gml("test.gml", destringizer=int)                           
    >>> J.nodes                                                                 
    NodeView((0, 1, 2, 3))
```

Reply:
Ok, see latest commit.